aqt/metadata.py

```diff
@@ -695,26 +695,33 @@ def describe_filters(self) -> str:
         return "{} with minor version {}".format(self.archive_id, self.filter_minor)
 
 
-def suggested_follow_up(meta: MetadataFactory, printer: Callable[[str], None]) -> None:
+def suggested_follow_up(meta: MetadataFactory) -> List[str]:
```

Comment:
It can be `print_suggested_follow_up()` and accept log level such as `logging.INFO` and `logging.ERROR`.
You can use logger as follows;
```
logger = logging.getLogger()
logger.log(logging.INFO, msg)
```
https://docs.python.org/ja/3/library/logging.html#logging.Logger.log

This allow you not to pass Callable.

Reply:
How about calling it `log_suggested_follow_up(meta, log_level)`, since making that change would restrict it to outputting to the logger?

Reply:
After making this change, the test `test_log_suggested_follow_up` passes when it is run on its own, but it fails when run with the rest of the tests. The error message says it failed because the logger file was closed while it was being written to: https://github.com/miurahr/aqtinstall/runs/3099834773?check_suite_focus=true#step:6:273.  

I think this is related to https://github.com/pytest-dev/pytest/issues/5502. That issue mentions some workarounds, but I haven't been able to get them to work.

Reply:
OK, I'd like to merge it with current version.